common/production_types.txt

```diff
@@ -841,22 +841,22 @@ artisan_dye = {
 
 ##### ARTISAN ONLY #####
 
-artisan_horse_to_food = {
```

Comment:
Note that these artisan cycles are there to help keep the economy moving -- I put these in at the recommendation of the developers of IgorPuir, who have a much better understanding of the economy than I do. We will almost certainly want to put them back